EventListener/ControllerListener.php

```diff
@@ -50,13 +50,18 @@ public function __construct(Reader $reader)
      */
     public function onKernelController(FilterControllerEvent $event)
     {
-        if (!is_array($controller = $event->getController())) {
+        if (!is_array($controller = $event->getController()) && !is_object($controller)) {
```

Comment:
closures are objects and defines an `__invoke` method. Of course there won't be any annotations, but is it worth it to avoid passing into this listener for such cases ? As you said, it's unlikely to happen, and it is actually "properly" handled (won't break).

Or maybe I missed the point ?